voluncheer/.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
```

Comment:
This file is redundant with the .gitignore in the root of the repo and can be removed.